js/watch/src/ui/styles/player.css

```
@@ -0,0 +1,148 @@
+.player:not(.player--chrome) {
+	cursor: none;
+}
+
+/*
+ * Fullscreen: fill the screen and center the video. We fullscreen this shadow
+ * container (not the bare <moq-watch>) so the overlay chrome stays visible.
+ * `.player--pseudo-fullscreen` is the iPhone fallback where the fullscreen API
+ * is unavailable; it pins the player to the viewport via fixed positioning.
+ */
+.player:fullscreen,
+.player:-webkit-full-screen,
```

Stylelint (17.12.0)

[error] Vendor-prefixed selector ":-webkit-full-screen" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)